pkg/apis/config/v1alpha1/types.go

```diff
@@ -55,3 +58,13 @@ type ACME struct {
 	// +optional
 	PropagationTimeout *metav1.Duration `json:"propagationTimeout,omitempty"`
 }
+
+// Custom contains optional custom configuration for the cert-controller-manager deployment
+type Custom struct {
+	// PrecheckNameservers is the value for the `--issuer.precheck-nameservers` command line parameter
+	// +optional
+	PrecheckNameservers *string `json:"precheckNameservers,omitempty"`
```

Comment:
Can you add an input validation? I suppose only conformant IP addresses are allowed here.

Reply:
validation added